ExtensionHandler/Linux/src/AzureRM.py

```diff
@@ -5,6 +5,9 @@
 #3. get command execution log summary
 #4. api versions merge and everywhere
 
+from __future__ import absolute_import
+from __future__ import print_function
```

Comment:
Are those two imports needed?

Reply:
Added them because the examples from aria said so. I removed them and checked, the logger in the current form doesn't need them and is working fine so these imports can be removed. However there might be a requirement to add these imports later if we develop the telemetry further.